app/jobs/participant_outcomes/batch_send_latest_outcomes_job.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ParticipantOutcomes
+  class BatchSendLatestOutcomesJob < ApplicationJob
+    queue_as :participant_outcomes
+
+    DEFAULT_BATCH_SIZE = 200
```

Comment:
Our estimate for how many will be required to send by March is around 14k+, so it will take substantially longer. However Im happy to wait for when we test on pre-prod to ensure how much we can bump up though, hopefully to at least change the delay to 1 minute